src/Row/index.stories.js

```diff
@@ -141,7 +141,7 @@ SectionHeaderExample.parameters = {
 
 export const JustifyingContent = () => (
   <div className="nds-typography">
-    <Row justifyContent="end">
+    <Row justifyContent="end" alignItems="center">
```

Comment:
now that `<Button type="plain"` no longer has a min height, update code example to vertically align different button types